res/values/strings.xml

```diff
@@ -679,6 +679,12 @@
     <string name="qrscan_contains_url">Scanned QR code URL:\n\n%1$s</string>
     <string name="qrscan_fingerprint_label">Fingerprint</string>
     <string name="qrscan_x_verified_introduce_myself">%1$s verified, introduce myself…</string>
+    <string name="withdraw_verifycontact_explain">This QR code can be scanned by others to setup a contact with you.\n\nIf you do not want that any longer, you can withdraw it. You can revive it by scanning it again.</string>
+    <string name="withdraw_verifygroup_explain">This QR code can be scanned by others to join the group \"%1$s\".\n\nIf you do not want that any longer, you can withdraw it. You can revive it by scanning it again.</string>
+    <string name="withdraw_qr_code">Withdraw QR code</string>
+    <string name="revive_verifycontact_explain">This QR code could be scanned to setup a contact with you, but was withdrawn. Currently the QR code is not functional.</string>
+    <string name="revive_verifygroup_explain">This QR code could be scanned by others to join the group \"%1$s\", but was withdrawn. Currently the QR code is not functional.</string>
```

Comment:
Actually, I think that it's enough to either write "but was withdrawn" or "currently the QR code is not functional", because that's saying the same thing, and users tend to like shorter dialogs

Reply:
'but was withdrawn' is the explanation for 'currently the QR code is not functional'. From my point of view, that's not exactly the same. It would be probably nicer to show this causality within one sentence.

Reply:
Not sure if I understood correctly: You propose to leave it as is?

Reply:
```suggestion
    <string name="revive_verifycontact_explain">Scanning this withdrawn QR code will no longer set up contact with you.</string>
    <string name="revive_verifygroup_explain">Scanning this withdrawn QR code will no longer allow joining the \"%1$s\" group.</string>
```

Reply:
I like this proposal

Reply:
In an a/v with @cyBerta we said that we would like to use the longer version that also explains the context to the user.

Reply:
Why?